payment_alipay/README.rst

```diff
@@ -0,0 +1,57 @@
+
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Alipay Payment Acquirer
+=======================
+
+This module extends the functionality of Payment to add a new payment acquirer Alipay.
+
+Installation
+============
+
+To install this module, you need to:
+
+ * have basic modules installed (portal_sale, website_sale)
+
+Usage
+=====
+
+* configure website payment method on `Sales > Configuration > Sales > Payment Methods`.
+
```

Comment:
a more detailed procedure for the setup is appreciated (screenshots are nice to have but always useful)